src/jenkins.py

```diff
@@ -212,14 +235,18 @@ def _unlock_wizard(connectable_container: ops.Container) -> None:
     )
 
 
-def _install_config(connectable_container: ops.Container) -> None:
+def _install_configs(connectable_container: ops.Container) -> None:
     """Install jenkins-config.xml.
 
     Args:
         connectable_container: The connectable Jenkins workload container.
     """
     with open("templates/jenkins-config.xml", encoding="utf-8") as jenkins_config_file:
         connectable_container.push(CONFIG_FILE_PATH, jenkins_config_file, user=USER, group=GROUP)
+    with open("templates/jenkins.yaml", encoding="utf-8") as jenkins_casc_config_file:
+        connectable_container.push(
+            JCASC_CONFIG_FILE_PATH, jenkins_casc_config_file, user=USER, group=GROUP
+        )
```

Comment:
There's a method called `push_path` in `Container` class that can directly push a local file to the workload container.

Reply:
There currently seems to be a bug when using `push_path`, https://github.com/canonical/operator/issues/948 which causes the unit tests to fail. When the issue gets merged, I can change it! Thanks for the review!